spring-jdbc/src/main/java/io/crate/spring/jdbc/samples/domain/User.java

```diff
@@ -0,0 +1,49 @@
+package io.crate.spring.jdbc.samples.domain;
+
+public class User {
```

Comment:
Same here, maybe we could make this class immutable ?

Reply:
Why?